src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -45,7 +47,7 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
 
         private void WriteToolInfo(string name, string fileVersion, Version assemblyVersion)
         {
-            _writer.WriteObjectStart("toolInfo");
+            _writer.WriteObjectStart("tool");
             _writer.Write("name", name);
             _writer.Write("version", assemblyVersion.ToString(fieldCount: 3));
```

Comment:
re 'semanticVersion', that's new, eh? I didn't see it. The way we write it now was conformant to prior iteration where 'version' was supposed to be semantic